minicluster/src/main/java/org/apache/accumulo/cluster/RemoteShellOptions.java

```diff
@@ -69,8 +70,8 @@ public RemoteShellOptions() {
       if (f.exists() && f.isFile() && f.canRead()) {
         FileReader reader = null;
         try {
-          reader = new FileReader(f);
-        } catch (FileNotFoundException e) {
+          reader = new FileReader(f, UTF_8);
+        } catch (IOException e) {
```

Comment:
It's odd that this constructor throws a more general exception, I would think it would have been the other way around.

Reply:
This constructor is newer, so they probably made it less specific because they declared their exceptions too narrowly when they wrote the original method.

Reply:
Ah OK, that makes sense.